src/slides/slides.css

```diff
@@ -534,14 +543,16 @@ body:lang(en) {
   z-index: 1000;
 }
 
-.reveal .copyright {
+.reveal .slides footer.copyright {
   left: 0px;
   width: 100%;
   text-align: center;
+  font-size: 70%;
+  color: var(--grey);
 }
 
-.reveal .slide-number {
-  right: 30px;
+.reveal .slides footer.slide-number {
```

Comment:
I would lower the font size too like for copyright.

Reply:
OK, done